shared/js/background/components/dashboard-messaging.js

```diff
@@ -69,7 +69,36 @@ export default class DashboardMessaging {
         if (!tab) {
             return;
         }
-        const pageParams = (await browser.tabs.sendMessage(tab.id, { getBreakagePageParams: true })) || {};
+
+        // Try to get breakage data from content-scope-scripts, with fallback to legacy
+        let pageParams = {};
+        try {
+            // Send message to content-scope-scripts via content script
+            // Note: must use messageType format for content-scope-scripts messaging
+            await browser.tabs.sendMessage(tab.id, {
+                messageType: 'getBreakageReportValues',
+            });
+
+            // Wait briefly for the breakageReportResult handler to receive and store data
```

Comment:
This timeout won't work as you're probably expecting, you're awaiting the `sendMessage` so it will block anyway, you're just awaiting a 500ms time here.